src/main/scala/io/iohk/ethereum/keystore/Wallet.scala

```diff
@@ -9,6 +9,6 @@ import org.bouncycastle.crypto.AsymmetricCipherKeyPair
 case class Wallet(address: Address, prvKey: ByteString) {
   lazy val keyPair: AsymmetricCipherKeyPair = keyPairFromPrvKey(prvKey.toArray)
 
-  def signTx(tx: Transaction, chainId: Option[Byte]): SignedTransaction =
+  def signTx(tx: Transaction, chainId: Option[Byte]): (SignedTransaction, Address) =
```

Comment:
Why are you returning a tuple if you defined SignedTransactionWithSender case class? If you use this class it will be more readable - you won't need to use _._x syntax.

Reply:
ahh good point, i defined this case class in somewhat different context and later forgot that i have here ready. I will change it.